### PR TITLE
Test for 201, shell script for executing on docker, skip few tests for agent of org

### PR DESCRIPTION
**Pull request by payal-bonde**
- test to cover 201 scenario of private EORI kickout screen
- temporary changed to tags to skip execution of known issues
 - added shell script to run on dockerized chrome

### CODE REVIEW

**Comment by payal-bonde** [comment]
I have purposely changed tags for few tests as we need clarity on Agent for Org journey and want to skip execution till decision is made and flow is fixed for the journey. Ket me know if there are concerns

**Review by Ewan-James-Donovan — APPROVED** [review]
LGTM

**Review by paulhargrave — APPROVED** [review]

File: src/test/resources/features/AgentsCredTrue/AgentActingOnBehalfOrganisationAssistant.feature [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @OrganisationAssistantAgentsCred
-@EndToEndJourneyAgentsCred
+#@EndToEndJourneyAgentsCred
+@WIPFail
```
Comment (C-Thomp) [inline src/test/resources/features/AgentsCredTrue/AgentActingOnBehalfOrganisationAssistant.feature:3]: do we want this tag?

Comment (payal-bonde) [reply src/test/resources/features/AgentsCredTrue/AgentActingOnBehalfOrganisationAssistant.feature:3]: Yes, so these are the tests for agent org journey that are under discussion. Till then they these tests will fail and it's better to not execute them and have them included again once the fix is done

**Review by C-Thomp — APPROVED** [review]
👍

**Pull request merged**